src/models.js

```diff
@@ -300,36 +321,24 @@ module.exports = ({ cooler, isPublic }) => {
     },
   };
 
-  const isPost = (message) =>
-    lodash.get(message, "value.content.type") === "post" &&
-    lodash.get(message, "value.content.text") != null;
-
   const isLooseRoot = (message) => {
     const conditions = [
       isPost(message),
-      lodash.get(message, "value.content.root") == null,
-      lodash.get(message, "value.content.fork") == null,
+      hasNoRoot(message),
+      hasNoFork(message),
     ];
 
     return conditions.every((x) => x);
```

Comment:
Is there a reason to use `conditions.every` here, instead of something like:

`return isPost(message) && hasNoRoot(message) && hasNoFork(message)`

Reply:
Nope, I think I was trying to be clever and/or assuming that there would be tons of conditions.